### ISSUE TITLE
Audio Example Request

### ISSUE THREAD

**Issue by toxicated101** [alexa-js/alexa-app#245]
Would it be possible to create an Audio Example, I know there are chunks in the documentation but I think an example that had two audio files  and would play one after the other would be a good starting block.

**Comment by dblock** [comment]
Please contribute.

I have a podcast intent that plays audio [here](https://github.com/artsy/elderfield/blob/master/functions/artsy/index.js#L298) that you can look at.

**Comment by toxicated101** [comment]
It works from an Amazon Echo Dot even though it appears to stop on the 3rd track with a  400 error.

Thanks

**Comment by supriya2502** [comment]
@dblock Thanks a lot for the script , Is it possible to say thank you when the audio finishes ?

**Comment by dblock** [comment]
Not AFAIK, I believe `res.say` will play before, but I could be wrong.

**Comment by lazerwalker** [comment]
Yeah, that tracks with my experience as well. Playing an audio stream is essentially entering a separate mode that means your skill gives up control of playback and control flow.
